profiles/consent-state/terraform/consent/cloudsql-init.tf

```diff
@@ -0,0 +1,52 @@
+resource "google_sql_ssl_cert" "client_cert" {
```

Comment:
Do you think this is a candidate for becoming a module? (not in this PR)

Reply:
Mmm sure I can see that